config/initializers/z_plugin_parsers.rb

```diff
@@ -0,0 +1,3 @@
+# Ansible
+Foreman::Plugin.fact_importer_registry.register(:ansible, ForemanAnsible::StructuredFactImporter, false)
+FactParser.register_fact_parser :ansible, AnsibleFactParser, false
```

Comment:
So this is intentionally moved to the end of initializer chain in Rails so every existing fact parser by a plugin is overridden with this.

It could be useful to change `register_fact_parser` method to issue a warning into logger if there already was an existing parser registered to a particular name and which class was that. So we know if this happens for some users who would e.g. forget to upgrade their plugins for any reason.